src/mpid/ch4/src/ch4i_workq.h

```diff
@@ -134,7 +134,6 @@ MPL_STATIC_INLINE_PREFIX void MPIDI_workq_pt2pt_enqueue(MPIDI_workq_op_t op,
         case IPROBE:
             {
                 struct MPIDI_workq_iprobe *wd = &pt2pt_elemt->params.pt2pt.iprobe;
-                wd->count = count;
```

Comment:
I agree with this, but would squash with the commit that removes the datatype. I can do so when signing off.

Reply:
Done with the latest push.